### ISSUE TITLE
[GRAPHQL] mutations? 

### ISSUE THREAD

**Issue by Zignature** [ExpressionEngine/Coilpack#14]
Is it possible to perform mutations with GraphQL? 
Basically I'd like to create a CRUD app with a Nuxt frontend.

If at all possible, how would I go about setting up authentication (in Nuxt)?
I'd prefer EE authentication so I can maintain members in my EE database.

I am the only one, and will remain the only one, that has access to the control panel. My coworkers, though very capable in doing their job, are all digital illiterates. I do not want them to screw up my EE installation 🤣

**Comment by Zignature** [comment]
I found out mutations aren't supported (yet?).

**Comment by bryannielsen** [comment]
@Zignature yes sorry currently we only support querying data through GraphQL.  We have talked about supporting mutations through Coilpack but that feature isn't on the timeline just yet.

**Comment by Zignature** [comment]
Okidoki 👍🏽 
I'll create a separate dashboard with EE for the coworkers 😏
